src/CryptoBank.WebAPI/Features/Auth/Errors/AuthLogicConflictErrorCode.cs

```diff
@@ -0,0 +1,11 @@
+namespace CryptoBank.WebAPI.Features.Auth.Errors;
+
+public static class AuthLogicConflictErrorCode
+{
+    private const string Prefix = "auth_";
+
+    public const string RefreshTokenEmpty = Prefix + "refresh_token_empty";
```

Comment:
This is an interesting situation. I thought about these error codes like it is possible to reuse them everywhere in the application. But I chose a too-narrow name. Maybe If I use the name `AuthErrorCodes` it will be better?

Also, I think `RefreshTokenDoesNotExist` is a logic conflict error because it is a step in the process of verification, and if I validate it in `RequestValidator` there are two operations of getting a token from DB.

Reply:
I separated validation error codes and logic conflict error codes. Also, I made them type-safe, now it is impossible to use validation error code in `LogicConflictException`.  What do you think about this approach?